src/_pytest/debugging.py

```diff
@@ -274,13 +276,24 @@ def pytest_pyfunc_call(self, pyfuncitem):
 def _test_pytest_function(pyfuncitem):
     _pdb = pytestPDB._init_pdb("runcall")
     testfunction = pyfuncitem.obj
-    pyfuncitem.obj = _pdb.runcall
-    if "func" in pyfuncitem._fixtureinfo.argnames:  # pragma: no branch
-        raise ValueError("--trace can't be used with a fixture named func!")
-    pyfuncitem.funcargs["func"] = testfunction
-    new_list = list(pyfuncitem._fixtureinfo.argnames)
-    new_list.append("func")
-    pyfuncitem._fixtureinfo.argnames = tuple(new_list)
+
+    # we need a copy of bdb.runcall because it takes `func` as a parameter
+    # which means we can't have a kwarg called func
```

Comment:
Ah.. in 3.8 it is positional-only (https://github.com/python/cpython/blob/9e87d5d964f4f49c24dc3dd4bb7e84ec8db1876e/Lib/bdb.py#L621).

I'm not sure why we need to support kwargs named "func" for the param, when pytest does not allow for them to be kwargs in the first place?  ("function already takes an argument 'param' with a default value")
```python
import pytest


@pytest.mark.parametrize("param", [1, 2])
def test_func_kw(param="foo"):
    pass
```